app/etc/config.php

```diff
@@ -329,16 +329,21 @@
         'Magento_WishlistGraphQl' => 1,
         'Checkout_Customize' => 1,
         'Mage2tv_Js' => 1,
+        'MagePal_Core' => 1,
+        'MagePal_GmailSmtpApp' => 1,
         'MyCompany_ExampleAdminNewPage' => 1,
         'PayPal_Braintree' => 1,
         'PayPal_BraintreeGraphQl' => 1,
+        'Tasks_BlogManager' => 1,
```

Comment:
@gayathri-aravind What's this module about?

Reply:
This is the module I created for learning purpose. It is not related to Feedback module.